src/main/java/seedu/task/model/util/SampleDataUtil.java

```diff
@@ -12,12 +12,12 @@
     public static Task[] getSampleTasks() {
         try {
             return new Task[] {
-                new Task(new Description("Alex Yeoh"), null, null, new UniqueTagList("friends")),
-                new Task(new Description("Bernice Yu"), null, null, new UniqueTagList("colleagues", "friends")),
-                new Task(new Description("Charlotte Oliveiro"), null, null, new UniqueTagList("neighbours")),
-                new Task(new Description("David Li"), null, null, new UniqueTagList("family")),
-                new Task(new Description("Irfan Ibrahim"), null, null, new UniqueTagList("classmates")),
-                new Task(new Description("Roy Balakrishnan"), null, null, new UniqueTagList("colleagues"))
+                new Task(new Description("Alex Yeoh"), null, null, new UniqueTagList("friends"), null),
```

Comment:
It looks like you've made the `id` field non-nullable in the constructor (which seems right to me). Will this cause tests to fail?